### PR TITLE
refactor(services): add kind and params

### PR DESCRIPTION
**Pull request by FlorentinDUBOIS**

### CODE REVIEW

File: app/models/repositories/ServiceRepository.scala [hunk 1]
```diff
@@ -60,6 +50,16 @@ class ServiceRepository @Inject()(dbapi: DBApi)(implicit ec: models.DatabaseExec
...
+  private def completeParser = parser[Service]() ~ parser[Check]()
+
+  private def gatherResults(lines: Seq[Service~Check]): Seq[CompleteService] = {
+    lines
+      .map({ case (s ~ e) => s -> e})
+      .groupBy(_._1)
+      .map({ case (s, vs) => CompleteService(s, vs.map(_._2))})
+      .toList
+  }
+
```
[the remark this answers is not in the document]
Comment (FlorentinDUBOIS) [reply app/models/repositories/ServiceRepository.scala]: The formatter was excited. I will remove this changes.

File: app/models/Scheduler.scala [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
       .groupBy(_._2 % buckets)
       .toList
       .map(_._2.map(x => x._1))
   }
+
+  sealed trait Tick {
+    def kind: Kind
+  }
+
+  case class HTTPTick() extends Tick {
```
Comment (urcadox) [inline app/models/Scheduler.scala]: These can be changed to case objects (if you remember, we were going to switch to case classes to put the kind parameter and then switched back to this but left the case classes).

Comment (FlorentinDUBOIS) [reply app/models/Scheduler.scala]: Tu ne peux pas hériter de Tick si c'est un case object :/

Comment (urcadox) [reply app/models/Scheduler.scala]: Yes you can, I just made sure.

This works:

```scala
trait Foo {
  def bar: String
}
case object Foobar extends Foo {
  val bar = "It works"
}
```

**Pull request merged**